package.json

```diff
@@ -74,7 +74,7 @@
     "@types/ws": "^7.2.4",
     "@typescript-eslint/eslint-plugin": "^4.4.0",
     "@typescript-eslint/parser": "^4.4.0",
-    "@web/test-runner": "^0.8.4",
+    "@web/test-runner": "^0.9.2",
```

Comment:
the old version started timing out on CI...upgrading to this fixes it. Not entirely sure why!